### PR TITLE
Side filters

### PR DESCRIPTION
**Pull request by LSvt**
Uses the dream dataset with selectors.

Uses the dream dataset scraped from the website. Has selectors in the side menu to select datasets, gender, ... Unselected dreams are smaller.

### CODE REVIEW

File: frontend/js/bubble_chart.js [hunk 1]
```diff
@@ -272,9 +275,10 @@ function bubbleChart() {
    */
   function moveToYears(alpha) {
     return function (d) {
-      var target = yearCenters[d.year];
+      //TODO
```
Comment (kaizensparc) [inline frontend/js/bubble_chart.js]: ??

File: frontend/js/bubble_chart.js [hunk 2]
```diff
@@ -396,8 +419,54 @@ function setupButtons() {
...
+      } else {
+        button.classed('selecting', true);
+        button.classed('active', false);
+      }
+
+      // Select or unselect the corresponding nodes
+      myBubbleChart.selectNodes();
+    });
+
+    //TODO: make this compact
```
Comment (kaizensparc) [inline frontend/js/bubble_chart.js]: ??

File: frontend/js/bubble_chart.js [hunk 3]
```diff
@@ -396,8 +419,54 @@ function setupButtons() {
...
+    .on("input", function() {
+          // Select or unselect the corresponding nodes
+            myBubbleChart.selectNodes();
+    });
+    d3.select('#yearMax')
+    .on("input", function() {
+          // Select or unselect the corresponding nodes
+            myBubbleChart.selectNodes();
+    });
+    /*d3.selectAll(".inputRange")
```
Comment (kaizensparc) [inline frontend/js/bubble_chart.js]: Dead code here

File: frontend/js/bubble_chart_old.js [hunk 4]
```diff
@@ -0,0 +1,476 @@
+//Charte graphique
```
Comment (kaizensparc) [inline frontend/js/bubble_chart_old.js]: Dead file ?

**Review by kaizensparc — CHANGES_REQUESTED** [review]
?? Means 'I don't understand, could you explain?'
Dead file to remove (if it's in fact a dead file)

**Comment by LSvt** [comment]
Removed dead code to address requested changes. Coded the order by year function.

**Review by kaizensparc — APPROVED** [review]
lgtm

**Pull request merged**